core/src/main/java/cucumber/runtime/order/OrderType.java

```diff
@@ -0,0 +1,26 @@
+package cucumber.runtime.order;
+
+import java.util.List;
+import java.util.Map;
+
+import gherkin.events.PickleEvent;
+
+public abstract class OrderType {
+
+	public static final String NONE_ORDER_TYPE = "None";

```

Comment:
Type names should be lower case or case insensitive.

Reply:
Will change it.